spec/models/manageiq/providers/kubernetes/container_manager/refresh_parser_spec.rb

```diff
@@ -3,6 +3,39 @@
 describe ManageIQ::Providers::Kubernetes::ContainerManager::RefreshParser do
   let(:parser)  { described_class.new }
 
+  describe "parse_namespaces" do
+    it "handles simple data" do
+      expect(parser.send(
+               :parse_namespaces,
```

Comment:
You mean:
```
      expect(parser.send(
        :parse_namespaces,
```
?  Rubocop doesn't like that (it doesn't like many existing examples in this file either).
It says "Indent the first parameter one step more than parser.send(." which by trial and error I found to mean 2 spaces deeper than `parser`.